Xamarin.Forms.Xaml/XamlServiceProvider.cs

```diff
@@ -91,19 +92,63 @@ public void Add(Type type, object service)
 
 	class XamlValueTargetProvider : IProvideParentValues, IProvideValueTarget
 	{
+		
+		Lazy<object> targetProperty;
 		public XamlValueTargetProvider(object targetObject, INode node, HydratationContext context, object targetProperty)
```

Comment:
the targetProperty should come from the caller, not induced by this method. Also, this code ignore the parameter value.